lib/rules/padding-line-between-statements.js

```diff
@@ -472,7 +497,8 @@ module.exports = {
 
         messages: {
             unexpectedBlankLine: "Unexpected blank line before this statement.",
-            expectedBlankLine: "Expected blank line before this statement."
+            expectedBlankLine: "Expected blank line before this statement.",
+            expectedCount: "Expected {{blankLine}} blank lines. Found {{count}}."
```

Comment:
I think this message should also have "before this statement"

Reply:
@kaicataldo @mdjermanovic, this was the case, my error message was corrected, which option is better?